### ISSUE TITLE
datasets

### ISSUE THREAD

**Issue by xpLF1314** [Chasel-Tsui/mmrotate-dcfl#24]
Hello. I want to use the config file"dotav2_test.py" to train the model, but in this file, I need the paths of the training set, the validation set, and the test set. But through the readme file of the data set, by running the image_split.py file, I can only get the training set and the test set. So, do I need to go through ss_val.json to get the validation set?
